lib/nuca_backend_web/views/hotspot_view.ex

```diff
@@ -1,3 +1,25 @@
 defmodule NucaBackendWeb.HotspotView do
   use NucaBackendWeb, :view
+  alias NucaBackendWeb.HotspotView
+
+  def render("index.json", %{hotspots: hotspots}),
+    do: render_many(hotspots, HotspotView, "hotspot.json")
+
+  def render("hotspot.json", %{hotspot: hotspot}),
+    do: hotspot
+
+  def render("hotspot_details.json", %{hotspot: hotspot}) do
+    sterilized_cats = Enum.filter(hotspot.cats, fn cat -> cat.is_sterilized == true end)
+    unsterilized_cats = Enum.filter(hotspot.cats, fn cat -> cat.is_sterilized == false end)
```

Comment:
No need to filter twice - we can filter by one criteria and subtract the filtered list from the whole result set.